src/api/rpc_api/mod.rs

```diff
@@ -23,3 +24,7 @@ pub mod pallet_balances;
 pub mod pallet_transaction_payment;
 pub mod state;
 pub mod subscribe_events;
+
+pub(crate) fn extrinsic_has_failed(event_details: &EventDetails) -> bool {
+	event_details.pallet_name() == "System" && event_details.variant_name() == "ExtrinsicFailed"
+}
```

Comment:
Why does this not go into the `ac_node_api`?

Reply:
good point, let me try that